sdk/textanalytics/azure-ai-textanalytics/samples/authoring/sample_import_project.py

```diff
@@ -0,0 +1,44 @@
+# coding=utf-8
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+"""
+FILE: sample_import_project.py
+DESCRIPTION:
+    This sample demonstrates how to import a project, using a dummy project provided at https://docs.microsoft.com/en-us/rest/api/language/text-analysis-authoring/import?tabs=HTTP.
+USAGE:
+    python sample_import_project.py
+    Set the environment variables with your own values before running the sample:
+    1) AZURE_TEXT_AUTHORING_ENDPOINT             - endpoint for your Text Analysis resource.
+    2) AZURE_TEXT_AUTHORING_KEY                  - API key for your Text Analysis resource.
+    3) AZURE_TEXT_AUTHORING_STORAGE              - name of the storage container for the project to be imported into
+"""
+from multiprocessing import dummy
+
+
+def sample_import_project():
+    from azure.ai.textanalytics.authoring import TextAuthoringClient
+    from azure.core.credentials import AzureKeyCredential
+    from samples.authoring.dummy_project import dummy_project
```

Comment:
A customer is likely to have the project in JSON format. We should convert this to JSON and read in the file in the sample.